File: Tema1/ErrorRelativo.py
class ErrorRelativo:
    @staticmethod
    def calcular(valor_real, valor_medido):
        ea = abs(valor_real - valor_medido)
        return (ea / abs(valor_real)) * 100


if __name__ == "__main__":
    er = ErrorRelativo.calcular(2.50, 2.45)
    print(f"Error relativo: {er:.4f} %")

File: tema1/ErrorRelativo.java
/**
 * Calcula el error relativo porcentual.
 */
public class ErrorRelativo {
    /**
     * Calcula el porcentaje de error relativo respecto al valor real.
     * @param valorReal Valor exacto.
     * @param valorMedido Valor medido.
     * @return Porcentaje de error relativo.
     */
    public static double calcular(double valorReal, double valorMedido) {
        double ea = Math.abs(valorReal - valorMedido); // error absoluto
        return (ea / Math.abs(valorReal)) * 100;
    }

    public static void main(String[] args) {
        // Caso de prueba
        double errorRel = calcular(2.50, 2.45);
        System.out.printf("Error relativo: %.4f %%\n", errorRel);
    }
}
